src/main/kotlin/com/hedvig/accountservice/schedulers/web/ScheduleController.kt

```diff
@@ -203,18 +200,16 @@ class ScheduleController @Autowired constructor(
   fun chargeAllMembers(
     @RequestParam scheduledBy: String
   ): ResponseEntity<Void> {
-    val approvedMemberIds = stateManager.findAllMemberIds(ChargeStatus.APPROVED_FOR_CHARGE)
-    approvedMemberIds.forEach { memberId ->
-      val transition = commandGateway.sendAndWait<SchedulerStateTransition>(
-        ScheduleChargeCommand(
-          memberId = memberId,
-          scheduledBy = scheduledBy
-        )
-      )
-      if (transition.outcome != SchedulerStateTransitionOutcome.ALLOWED_TRANSITION) {
-        logger.error { transition.message }
-      }
-    }
+    scheduleChargesService.scheduleChargesForAll(scheduledBy)
+    return ResponseEntity.noContent().build()
+  }
+
+  @PostMapping("paymentProvider/{paymentProvider}/charges")
```

Comment:
I like it to be structured like this. Do as you please though!
```suggestion
  @PostMapping("charges/provider/{paymentProvider}")
```